src/common/components/templates/InfoBanner.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { FaTimes } from "react-icons/fa";
+import * as Toast from "@radix-ui/react-toast";
+
+export default function InfoToast() {
+  const [isDisplayed, setIsDisplayed] = useState(false);
+  const [message, setMessage] = useState("");
+  const [duration, setDuration] = useState(10000);
+  const router = useRouter();
+  const { pathname, query } = router;
+  const userFarcasterName = query.handle;
+
+  useEffect(() => {
+    let storedState;
+
+    if (pathname === "/homebase") {
+      storedState = localStorage.getItem("homebaseToastDisplayed");
+      if (!storedState && localStorage.getItem("privy:token") !== null) {
```

Comment:
We should use the `isSignedIn` store variable, not a check for the privy token to detect logged in status

Reply:
yeah, that was a hack to see if the user was logged in , I was not aware of the isSignedin variable, I didnt mess with the sign in process yet, will change !

Reply:
just replaced for that 
` ...  if (!storedState && isLoggedIn) { ...`